### PR TITLE
Fix go get cmd

### PR DESCRIPTION
**Pull request by meadlai**
Fix go get CMD for client

### CODE REVIEW

**Review by ptabor — APPROVED** [review]
Please change the commit (not PR) title to: 

```
"client: Update README.md" 
```

to have the tests PASS

**Review by ptabor — APPROVED** [review]

File: client/v3/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ## Install
 
 ```bash
-go get go.etcd.io/etcd/clientv3
+go get go.etcd.io/etcd/client/v3
```
Comment (spzala) [inline client/v3/README.md:11]: lgtm
@meadlai thanks, as @ptabor mentioned, please update commit title.

**Review by spzala — APPROVED** [review]

**Comment by xiang90** [comment]
@ptabor 

As we have more and more new comers contributing to etcd project, I feel we should just remove the strict commit title style requirement. As long as the commit title is clear, we should just merge it.

Do you agree? Can you help to remove the check? /cc @gyuho

**Comment by ptabor** [comment]
> @ptabor
> 
> As we have more and more new comers contributing to etcd project, I feel we should just remove the strict commit title style requirement. As long as the commit title is clear, we should just merge it.
> 
> Do you agree? Can you help to remove the check? /cc @gyuho

Will do. ETA: Mon.

**Pull request merged**